README.md

```diff
@@ -38,5 +42,23 @@ docker run \
 You can run it with `docker-compose up`. It uses `docker-compose.yaml` file.
 
 ### Serving generated files
+
 Build files are served by default at http://127.0.0.1:9009/
 Documentation is live (browser updates view each time any file changes).
+
+### Remote docs
+
+If You'd like to fetch docs from a remote repository, please add requested file to the `remote_docs_config.json`.
+Each key of the json file represents a desired directory. Then You need to provide remote `url` and `filename`.
+
+example:
+
+```json
+  "data_collection/web/frameworks/": {"url": "https://raw.githubusercontent.com/PiwikPRO/vue-piwik-pro/master/README.md", "filename": "Piwik_PRO_Library_for_Vue.md"}
+```
+
+To run the script You should have a projects container running (`docker compose up -d`). Then just run the script
+
+```bash
+docker exec -it ppms-publicdocs-ppms_public_docs-1 python3 download_remote_docs.py
+```
```

Comment:
This could be added to [Release Checklist](https://piwikpro.atlassian.net/wiki/spaces/PPDP/pages/86351642756/Release+checklist) and performed by one of Public Docs maintainer (wink wink @kororokke)